basic_module/distribution/src/lib.rs

```diff
@@ -58,7 +59,7 @@ impl IntermediateRewards {
 
         context.set(
             get_storage_id(),
-            &get_key(),
+            &StorageKey::default(),
```

Comment:
I think it's unnatural to utilize [0; 32] as a key for the entry `intermediate_reward`. As I [mentioned](https://github.com/CodeChain-io/foundry/pull/328#discussion_r414313470), just a simple descriptive key value is enough.

Reply:
So you mean, any type such as u16 or u64 is fine?